xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/less/general.less

```diff
@@ -162,10 +162,11 @@ body.content {
     /* UIs in content where we want to not underline by default.
       Those elements usually contain only links that are not inline (and a lot of links). */
     #xwikicontent {
-      &.xwiki-livetable-container,
-      &.xtree,
-      &.xwikitabbar,
-      &.buttonwrapper {
+      & .xwiki-livetable-container,
+      & .xtree,
+      & .xwikitabbar,
+      & .buttonwrapper,
+      & #user-menu-col {
```

Comment:
Disable account is still underlined in the user profile

![image](https://github.com/xwiki/xwiki-platform/assets/327856/8625e4ef-715b-4d60-8cb9-06c003fe7458)

cc @michitux for that part as you followed the topic more than I did.

Reply:
![image](https://github.com/xwiki/xwiki-platform/assets/28761965/0fe20e59-bedb-4018-b816-06d17e907b06)

Okay, as of now the action links (that often look like a button) are only addressed by `& .buttonwrapper` which takes care of all the fake buttons in [inline forms](https://www.xwiki.org/xwiki/bin/view/Documentation/DevGuide/FrontendResources/InlineForms/). 
As seen in the screenshot, this link is not contained in a `.buttonwrapper` so it's not matched with our selector.
An improvement would be to match on `.btn` itself. I'll keep track of this comment and start a fix PR with this fix along some other we might find along the way next week (to hopefully get it in the full release, should be very safe to merge :) ).